### PR TITLE
Tom/re structure project

### PR DESCRIPTION
**Pull request by tomyang729**
Related issue #4 #2

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -38,49 +39,9 @@ lazy val managementapp_play = (project in file("./managementapp-play"))
...
-  .disablePlugins(PlayLayoutPlugin)
-  .settings(
-    libraryDependencies += "com.trueaccord.scalapb" %% "compilerplugin" % "0.5.41",
-    PB.targets in Compile := Seq(
-      PB.gens.java -> ((sourceDirectory in Compile).value / "protobuf_generated")
-    )
-  )
\ No newline at end of file
+lazy val managementapp_models = (project in file("./models"))
+  .settings(commonBuildSettings: _*)
```
Comment (omzmarlon) [inline build.sbt]: ./models does not need slick dependencies? I thought we need the "Table" class from slick to define tables?

Comment (omzmarlon) [inline build.sbt]: ./models does not need slick dependencies? I thought we need the "Table" class from slick to define tables?

Comment (tomyang729) [reply build.sbt]: oh I thought models is only for defining schema classes or traits. But if ./models also depends on play and play-slick, why not just put models into ./play, like the sbt standard layout. Under play/src/main/com/managementapp we have /controllers and /models

Comment (omzmarlon) [reply build.sbt]: Sure we can do that for now

**Review by omzmarlon — CHANGES_REQUESTED** [review]
Remove the (project in file(./models)) for this project now and then merge the changes

And also check-in all the sql scripts you used for db creation please

按错键了😂你把这两个改一下就直接merge吧～

**Review by omzmarlon — APPROVED** [review]

**Pull request merged**